src/BenchmarkDotNet/Toolchains/DotNetCli/DotNetCliCommand.cs

```diff
@@ -62,7 +62,12 @@ public BuildResult RestoreThenBuild()
 
             var buildResult = BuildNoRestore();
             if (!buildResult.IsSuccess) // if we fail to do the full build, we try with --no-dependencies
+            {
+                Logger.WriteLineInfo(buildResult.StandardOutput);
+                Logger.WriteLineInfo(buildResult.StandardError);
+                Logger.WriteLineInfo("Build failed. Trying with --no-dependencies");
                 buildResult = BuildNoRestoreNoDependencies();
+            }
```

Comment:
With the logging that you have added in #1950 I think it's not worth logging the output again here. Especially since `BuildNoRestoreNoDependencies`. 
```suggestion
                buildResult = BuildNoRestoreNoDependencies();
```